demos/imagenet_classification_demo/grid_mat.hpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <monitors/presenter.h>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    explicit GridMat(int inputImgsCount,
+                     Presenter& presenter,
+                     const cv::Size maxDisp = cv::Size{1920, 1080},
+                     const cv::Size aspectRatio = cv::Size{16, 9},
+                     double targetFPS = 60
+                     ):
+                     size{std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.height)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.width))
+                                    ),
+                          std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.width)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.height))
+                                    )},
+                     currSourceID{0} {
+        int minCellSize = std::min(maxDisp.width / size.width, maxDisp.height / size.height);
+        cellSize = cv::Size(minCellSize, minCellSize);
+        rectangleHeight = presenter.graphSize.height;
+
+        for (int i = 0; i < size.width; i++) {
+            for (int j = 0; j < size.height; j++) {
+                cv::Point p;
+                p.x = cellSize.width * i;
+                p.y = rectangleHeight + (cellSize.height * j);
+                points.push_back(p);
+            }
+        }
+
+        outImg.create((cellSize.height * size.height) + rectangleHeight, cellSize.width * size.width, CV_8UC3);
+        outImg.setTo(0);
+    }
+
+    cv::Size getSize() {
+        return size;
+    }
+
+    void listUpdate(std::list<std::tuple<cv::Mat, std::string, bool>>& imageInfos) {
+        if (!imageInfos.empty()) {
+            updateList.splice(updateList.end(), imageInfos);
+        }
+    }
+
+    void textUpdate(double avgFPS, double avgLatency, double accuracy,
+                    bool isFpsTest, bool showAccuracy,
+                    Presenter& presenter) {
+        // Draw a rectangle
+        cv::Point p1 = cv::Point(0, 0);
+        cv::Point p2 = cv::Point(outImg.cols, rectangleHeight);
+        rectangle(outImg, p1, p2, cv::Scalar(0, 0, 0), cv::FILLED);
+
+        presenter.drawGraphs(outImg);
+        
+        // Show FPS
+        double fontScale = 1.5;
+        int thickness = 2;
+        int baseline = 0;
```

Comment:
```suggestion
        int baseline;
```
`getTextSize()` will provide the value. More over you don't ever care about the value.